From: Debian Java Maintainers <pkg-java-maintainers@lists.alioth.debian.org>
Date: Sun, 12 Nov 2017 01:04:50 +0100
Subject: relax_warnings

resource-http produces lots of warnings and "-Werror" makes them errors.
---
 gradle/strictCompile.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gradle/strictCompile.gradle b/gradle/strictCompile.gradle
index b070602..2c2f9d4 100644
--- a/gradle/strictCompile.gradle
+++ b/gradle/strictCompile.gradle
@@ -17,7 +17,7 @@
 allprojects {
     ext.strictCompile = {
         tasks.withType(AbstractCompile) {
-            options.compilerArgs << "-Werror" << "-Xlint:all" << "-Xlint:-options" << "-Xlint:-serial"
+            options.compilerArgs << "-Xlint:all" << "-Xlint:-options" << "-Xlint:-serial"
         }
     }
     ext.strictCompileIgnoreDeprecations = {
